
---
title: """""""""""'Protocol for Construction of Genome-Wide Epistatic SNP Networks Using WISH-R Package.'"""""""""""
categories: 
    - 科学期刊
    - Stork 文献鸟订阅 - 关键词
author: Stork 文献鸟订阅 - 关键词
comments: false
date: 2021-03-22 04:47:59
thumbnail: ''
---

<div>   
Epistasis is the interaction between genes or genetic variants (such as Single Nucleotide Polymorphisms or SNPs) that influences a phenotype or a disease outcome. Statistically and biologically, significant evidence of epistatic loci for several traits and diseases is well known in human, animals, and plants. However, there is no straightforward way to compute a large number of pairwise epistasis among millions of variants along the whole genome, relate them to phenotypes or diseases, and visualize them. The WISH-R package (WISH-R) was developed to address this technology gap to calculate epistatic interactions using a linear or generalized linear model on a genome-wide level using genomic data and phenotype/disease data in a fully parallelized environment, and visualize genome-wide epistasis in many ways. This method protocol chapter provides an easy-to-follow systematic guide to install this R software in computers on Win OS, Mac OS, and Linux platforms and can be downloaded from https://github.com/QSG-Group/WISH with a user guide. The WISH-R package has several inbuilt functions to reduce genotype data dimensionality and hence computational demand. WISH-R software can be used to build scale-free weighted SNP interaction networks and relate them to quantitative traits or phenotypes and case-control diseases outcomes. The software leads to integrating biological knowledge to identify disease- or trait-relevant SNP or gene modules, hub genes, potential biomarkers, and pathways related to complex traits and diseases.  
</div>
            